.env.sample

```
@@ -15,3 +15,4 @@ NEXT_PUBLIC_SUPABASE_URL=https://your-project.supabase.co
 
 # サーバーサイド専用（管理者ダッシュボードのRLS回避用）
 SUPABASE_SERVICE_ROLE_KEY=your-service-role-key
+DISABLE_AUTH=true
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)